bioio/tests/accepting-plugin/accepting_plugin/reader.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Optional, Tuple
```

Comment:
Whats the difference here from using the dummy plugin?

Reply:
the plugin name is a little strange to me.

Reply:
The dummy plugin raises an exception when you try to instantiate it (or use it). This one returns. I picked "accepting" because it accepts lots of file types and if you instantiate it with `BioImage(...)` it will accept any arguments. We can certainly rename in follow-up PRs.